src/object/trigger_manager.c

```diff
@@ -6847,12 +6879,25 @@ tr_rename_trigger (DB_OBJECT * trigger_object, const char *name, bool call_from_
 	      trigger->name = newname;
 	      db_make_string (&value, newname);
 	      newname = NULL;
-	      error = db_put_internal (trigger_object, TR_ATT_NAME, &value);
-	      if (error == NO_ERROR)
+
+	      db_make_string (&value, trigger->name);
+	      error = db_put_internal (trigger_object, TR_ATT_FULL_NAME, &value);
+	      if (error != NO_ERROR)
```

Comment:
It should be modified in the following way.
```
error = db_put_internal (trigger_object, TR_ATT_FULL_NAME, &value);
if (error == NO_ERROR)
{
    db_make_string (&value, sm_simple_name (trigger->name));
    error = db_put_internal (trigger_object, TR_ATT_NAME, &value);
}
```

Reply:
Okay, I'll change it.